### PR TITLE
Fix endpoint IDs in examples

### PR DESCRIPTION
**Pull request by kolenic-martin**
This PR fixes endpoint IDs in airnode-examples. Several IDs did not correspond with what the Admin CLI generated so this needed to be put in line with expectations.

### CODE REVIEW

File: .changeset/famous-planes-impress.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (Siegrift) [inline .changeset/famous-planes-impress.md:1]: Can you remove two of these files? One changeset is enough.

**Review by Siegrift — CHANGES_REQUESTED** [review]
Please rebase (or merge) with master because the `yarn.lock` should not be modified.

File: yarn.lock [hunk 2]
```diff
@@ -17599,4 +17599,8 @@ zksync-web3@^0.4.0:
 zod@^3.11.6:
   version "3.17.3"
   resolved "https://registry.yarnpkg.com/zod/-/zod-3.17.3.tgz#86abbc670ff0063a4588d85a4dcc917d6e4af2ba"
+<<<<<<< HEAD
```
Comment (Siegrift) [inline yarn.lock:17602]: Please revert these changes.

Comment (martinkolenic) [reply yarn.lock:17602]: Sorry about that. It was pulled from master though, do you think my yarn changed that in the few minutes I had it in my repo?

Comment (Siegrift) [reply yarn.lock:17602]: You had a conflict during rebasing and you didn't resolve it. You can just drop the changes in this file.

**Comment by Siegrift** [comment]
No problem, just update the changelog to point to your new handle.
